grafana-plugin/src/containers/RotationForm/RotationForm.tsx

```diff
@@ -421,6 +422,13 @@ export const RotationForm = observer((props: RotationFormProps) => {
     }
   }, [store.timezoneStore.selectedTimezoneOffset]);
 
+  useEffect(() => {
+    window.addEventListener('resize', onResize);
```

Comment:
hide modal on resize so that it's not accidentally moved into corners where it's not visible